content/guides/compute.md

```diff
@@ -4,3 +4,331 @@ title: Compute Engine documentation | ARGO
 
 # Compute Engine
 
+Ar-Compute-engine is the argo component responsible for performing various transformations and computations on the collected metric data in order to provide availability and reliability metrics. The results produced by the compute-engine are forwarded and stored to the configured datastore (MongoDB)
+
+Ar-Compute-engine uses the hadoop software stack for performing calculations on the metric data as map-reduce jobs. These jobs can also be run locally (single node mode) in the absence of a proper hadoop cluster. Under the hood the engine uses Apache Pig for map-reduce job submission and execution.
+
+The main engine's input is the metric data collected from the broker network. These files (according to the default configuration of ar-consumer component) reside to the `/var/lib/ar-consumer/` folder.
```

Comment:
Why do you mention the broker network? I would expect to find a description in this document of the metric data schema that the compute engine expects.